### ISSUE TITLE
update for 2.4r2

### ISSUE THREAD

**Issue by dmitryuv** [Woki-Leo/Carbonfiber-X-Axis#2]
Hi,

XY joints needs to be updated for 2.4r2, so most of the parts can be used form the original STLs (like carriage) and not from (outdated) MODs. 

Specifically:
- shafts are now a bit closer to the rail, since new belts design press fit them agains the rail carriage, shifting their original position closer. With your current design belts won't be parallel to the rail.
- I'm not sure it makes sense to have cut corners without middle cut (as in the original design). You took off some material at the corners, but added in the middle. By slicer estimations there are no savings at all (not even 1g). I propose revert to the original here.

I've added comparison of the both parts (original and modified) and highlighted the difference in the shaft positions.

Thanks
<img width="495" alt="Screenshot 2022-04-28 at 13 37 43" src="https://user-images.githubusercontent.com/2512131/165735803-74bc451f-961b-4243-b51a-46c4b70c5d16.png">
!

**Comment by Leo-Wolf-the-Engineer** [comment]
Hey,

i currently have no time to do the changes needed for v2.4r2.

The material in the center slot was added because the walls for the slot added more material than they saved (this was tested with 20% + 2 walls) and it takes away strength and stiffness. not shure why it was there in the first place.
The camfers were my personal stylistic choice.

Feel free to modify it to your liking and open a Pull request!

**Comment by dmitryuv** [comment]
I'm currently in the process of modifying r2 joints, but i'm working from original CAD files to since changing STL mesh won't have the same end-quality. Unfortunately I won't be able to re-create your changes, so if you're OK I can later make pull request that will only include holes adjustments and pins mod.

**Comment by Leo-Wolf-the-Engineer** [comment]
the CAD is right here in the Github. And please use the Pinned version of them.

**Comment by dmitryuv** [comment]
yes, the CAD version of the non-r2 joint :) it's much harder to move around core dimensions than expand few holes, that's why i took original to modify.
